tests/test_playbooks/content_export_info.yml

```diff
@@ -21,13 +21,6 @@
         incremental: false
         destination_server: "airgapped.example.com"
 
-- hosts: tests
-  collections:
-    - theforeman.foreman
-  gather_facts: false
-  vars_files:
-    - vars/server.yml
-  tasks:
```

Comment:
why are you deleting these? the tasks need to run against the `tests` "host", so they are properly recorded.

Reply:
hmm, maybe this was a rebase error? I'll look again